production/helm/loki/values.yaml

```diff
@@ -14,8 +14,13 @@ affinity: {}
 ## Deployment annotations
 annotations: {}
 
+# enable tracing for debug, need install jaeger and specify jaeger_agent_host
+traceEnabled: false
```

Comment:
I would prefer:

```
tracing:
  jaegerAgentHost:
```

Reply:
tracing would not have more configs i think, so just use jaegerAgentHost.